### PR TITLE
[WIP] CORE-159 - Waiting List Status

### PR DESCRIPTION
**Pull request by AxonC**

### CODE REVIEW

File: app/Models/Training/WaitingListAccount.php [hunk 1]
```diff
@@ -65,4 +66,19 @@ public function incrementPosition($value = 1)
 
         return $this->position;
     }
+
+    public function atcHourCheck()
+    {
+        // for a user in a waiting list, they should have > 12 hours controlled within the UK.
+        $controllingSessions = Atc::whereAccountId($this->account_id)
+            ->whereBetween('disconnected_at', [\Carbon\Carbon::parse('3 months ago'), now()])->isUk();
```
Comment (atoff) [inline app/Models/Training/WaitingListAccount.php]: I assume this might need a `->get()` or something here?

Comment (AxonC) [reply app/Models/Training/WaitingListAccount.php]: It seems to work without this... could you pull and try this for me just to confirm?

Comment (atoff) [reply app/Models/Training/WaitingListAccount.php]: Nevermind, looks like the `sum()` function also acts as a `get()`er

**Comment by AxonC** [comment]
PR merged approved by @CalumTowers via phone call.

**Pull request merged**